build/linux/installer/conf/fluent-bit-geneva-logs_infra.conf

```diff
@@ -18,6 +18,19 @@
     Exclude_Path ${AZMON_CLUSTER_LOG_TAIL_EXCLUDE_PATH}
     ${TAIL_THREADED}
 
+#${KubernetesFilterEnabled}[FILTER]
```

Comment:
Also add the alias  "#${KubernetesFilterEnabled} Alias geneva_kubernetes_filter" after Name . This will be useful for the metrics and troubleshooting.

Reply:
added